x-pack/plugins/security_solution/public/detection_engine/rule_management/api/api.ts

```diff
@@ -206,14 +206,22 @@ export interface BulkActionAggregatedError {
   rules: Array<{ id: string; name?: string }>;
 }
 
+export interface BulkActionAttributes {
```

Comment:
Thanks for adding these types and discriminating between successful and error responses 👍 

Only thing: I also added these types for the bulk edit response while working on the `skipped` rules PR. Do you think it makes sense to move them to the `/common` folder and then import them to both server and public?

https://github.com/elastic/kibana/pull/144461/files#diff-0bddfc1ba78df99099e714716ea1bb5d969c5d6da65946d2f1cbb964382f2405R34

Reply:
@jpdjere Yes, if the types are the same on the front end and server side, it makes sense to move them to the common folder. Ideally, we should put there a runtime io-ts schema, which could be used for response validation, and derive the type from it.

Reply:
Allright, if we get merged this before than my PR I can update it on my side and refactor.

Reply:
> Allright, if we get merged this before than my PR I can update it on my side and refactor.

Yes, as this one targets 8.6, it would be safer to do refactoring in the `main`. Thank you, @jpdjere 👍